packages/ui-badge/package.json

```diff
@@ -25,20 +25,21 @@
   "dependencies": {
     "@babel/runtime": "^7.9.2",
     "@instructure/console": "^7.3.1",
+    "@instructure/emotion": "^7.3.1",
     "@instructure/ui-color-utils": "^7.3.1",
     "@instructure/ui-position": "^7.3.1",
     "@instructure/ui-react-utils": "^7.3.1",
     "@instructure/ui-testable": "^7.3.1",
     "@instructure/ui-themeable": "^7.3.1",
```

Comment:
could this dependency be removed?

Reply:
`import { ThemeablePropTypes } from '@instructure/ui-themeable'` is still using it in the component (if you meant the ui-themeable dependency :) )